RiotSwiftUI/Modules/Room/TimelinePoll/Coordinator/TimelinePollCoordinator.swift

```diff
@@ -86,7 +86,9 @@ final class TimelinePollCoordinator: Coordinator, Presentable, PollAggregatorDel
     }
     
     func toPresentable() -> UIViewController {
-        return VectorHostingController(rootView: TimelinePollView(viewModel: viewModel.context))
+        let controller = VectorHostingController(rootView: TimelinePollView(viewModel: viewModel.context))
```

Comment:
Just a thought, but instead of the swizzling, would the following SwiftUI route work?
```suggestion
        let controller = VectorHostingController(rootView: TimelinePollView(viewModel: viewModel.context).ignoresSafeArea())
```

Reply:
Ah yes, sorry I forgot to mention this. I did try that out, too, but it didn't help oddly. There was a combination of `ignoresSafeArea` and `viewSafeAreaInsetsDidChange` that seemed to work but I got that into an infinite loop state once or twice so I abandoned that path.

Reply:
Yeah this would just make the SwiftUI view ignore the safe area I suppose, but it wouldn't ignore it's UIKit parent bounds.

Currently trying to think about some other way to fix this, it looks weird to me that the SO you linked got so little attention, feels like we're missing on something.
Is it possible to reproduce this with any SwiftUI view nested in the table ? Maybe we are doing something odd in the view's code that makes iOS go crazy.

Reply:
> Is it possible to reproduce this with any SwiftUI view nested in the table?

Yes, it is actually. I had tried with just a rectangle statically sized to 100x100 points and it showed the same issue. So I'm pretty sure the root cause is not in the SwiftUI view itself. It may be somewhere in the table view logic we've built around this though. The safe area insets were not constant and seemed to change depending on where on the screen the cell was when it was first rendered.

Reply:
If this [comment](https://github.com/vector-im/element-ios/issues/5326#issuecomment-1174667616) means that we actually don't retrieve the same safe area on the ```PollPlainCell``` frame at some point, I'm going to suppose we can't find a better fix.

Reply:
Looks like it's the only option we have 🤦‍♂️. I can see new apis like `UIHostingConfiguration` and `UIHostingControllerSizingOptions` introduced on iOS 16. If we can make this work with it, we can update it in the future.

Reply:
Yeah, I noticed those, too. So hopefully we can undo this mess in the future. 🤞

Thanks all!